noir/compiler/wasm/src/compile.rs

```diff
@@ -33,28 +32,25 @@ export type CompiledContract = {
     name: string;
     functions: Array<any>;
     events: Array<any>;
+    file_map: Record<number, any>;
+    warnings: Array<any>;
 };
 
 export type CompiledProgram = {
```

Comment:
Can you rename these types to align with `ProgramArtifact` and `ContractArtifact`? I think ideally we should have these be consistent across JS and Rust.

This is a preexisting issue but I think this PR is a good time to make the change tbh.